lib/core/src/animation/physics/inertia.rs

```diff
@@ -261,6 +276,53 @@ impl PhysicsProperties {
     pub fn set_drag(&mut self, drag:DragProperties) {
         self.data.borrow_mut().drag = drag;
     }
+
+    /// Safe accessor to modify `SimulationThresholds`.
+    pub fn mod_thresholds<F:FnOnce(&mut SimulationThresholds)>(&mut self, f:F) {
```

Comment:
I don't like `mod` as a shortcut for `modify`. My first association is "module". Please use the whole word or name differently (`map_thresholds`?)

Reply:
I am changing it `modify_thresholds`.